src/vue/forms/AssetForm.vue

```diff
@@ -141,10 +197,16 @@
       </div>
     </form>
   </form-stepper>
+
+  <loader
```

Comment:
@max-nes here is a little offtop.

I don't like that we prefer such ugly formatting
```
<component
  :prop-1="a"
  :prop-2="b"
/>
```

instead of amazing beautiful awesome super version:

```
<component :prop-1="a"
           :prop-2="b"
/>

Reply:
@quinn71 IMO looks like a shit :) let’s discuss it outta the PR, as it is not related to it. please resolve if aggree